syncd/Sequencer.cpp

```diff
@@ -0,0 +1,175 @@
+#include "Sequencer.h"
+#include "swss/logger.h"
+
+using namespace syncd;
+
+// Helper function to execute all ready responses in order
+// check how many consecutive responses were executed in sucession and log it
+Sequencer::SequenceStatus Sequencer::executeReadyResponses() {

```

Comment:
should  not run under lock since block new ecxcute ... ? 
but need to think if we lock each iteration - two treads can run  in parallel ...